modules/tasks/tasks/src/main/java/org/motechproject/tasks/domain/mds/task/TaskActionInformation.java

```diff
@@ -28,6 +28,7 @@
 public class TaskActionInformation extends TaskEventInformation {
 
     private static final long serialVersionUID = -132464255615128442L;
+    private static final int MAX_VALUE_LENGTH = 20000;
```

Comment:
In the ticket description there is info about 'unlimited', so I suppose we should set this as highest as possible. Mysql varchar value is lower, so please set to this value